plugins/com.amansprojects.starterpack.sdPlugin/src/main.rs

```diff
@@ -119,7 +127,28 @@ async fn main() {
 	)
 	.unwrap();
 
-	if let Err(error) = init_plugin(GlobalEventHandler {}, ActionEventHandler {}).await {
+	let enigo = match Enigo::new(&Settings {
+		release_keys_when_dropped: true,
```

Comment:
You can just derive the default for the whole `Settings` struct. `release_keys_when_dropped: true` is the default value (see https://github.com/enigo-rs/enigo/blob/cf10baddde4b267b858c94ff08f697572d26a532/src/lib.rs#L481).

I only briefly looked at the code, but this is better usage of the `Enigo` struct. Recreating it each time leads to a number of issues as it keeps track of important state.

Reply:
I never used enigo for programs that run a long time, so I am also not sure if there can be any problems that occur after hours or so. If they are recoverable, it surely makes sense to attempt to recreate it. There shouldn't be much that can go wrong though. If you do encounter problem like that, please let me know so I am aware of it.

As for my other comment, sure you can be explicit. I don't know the code base or the context of the code. I just thought I'd mention it.